### PR TITLE
renderers: Convert to es6.

### PR DESCRIPTION
**Pull request by linbingquan**
Clean up for https://github.com/mrdoob/three.js/issues/19986#issuecomment-674046275

### CODE REVIEW

File: src/renderers/WebGLCubeRenderTarget.js [hunk 1]
```diff
@@ -109,20 +109,21 @@ WebGLCubeRenderTarget.prototype.fromEquirectangularTexture = function ( renderer
...
-	mesh.material.dispose();
+		return this;
 
-	return this;
+	}
+
+}
 
-};
+WebGLCubeRenderTarget.prototype.isWebGLCubeRenderTarget = true;
```
Comment (marcofugaro) [inline src/renderers/WebGLCubeRenderTarget.js:127]: could you use the `Object.defineProperty()` pattern demonstrated in #20110? 🙏

File: src/renderers/WebGLMultisampleRenderTarget.js [hunk 2]
```diff
@@ -1,30 +1,27 @@
...
 		this.samples = source.samples;
 
 		return this;
 
 	}
 
-} );
+}
 
+WebGLMultisampleRenderTarget.prototype.isWebGLMultisampleRenderTarget = true;
```
Comment (marcofugaro) [inline src/renderers/WebGLMultisampleRenderTarget.js:25]: could you use the `Object.defineProperty()` pattern demonstrated in #20110? 🙏

File: src/renderers/WebGLRenderTarget.js [hunk 3]
```diff
@@ -81,15 +79,16 @@ WebGLRenderTarget.prototype = Object.assign( Object.create( EventDispatcher.prot
...
+	dispose() {
 
 		this.dispatchEvent( { type: 'dispose' } );
 
 	}
 
-} );
+}
 
+WebGLRenderTarget.prototype.isWebGLRenderTarget = true;
```
Comment (marcofugaro) [inline src/renderers/WebGLRenderTarget.js:92]: could you use the `Object.defineProperty()` pattern demonstrated in #20110? 🙏

File: src/renderers/WebGL1Renderer.js [hunk 4]
```diff
@@ -1,17 +1,15 @@
...
+		super( parameters );
 
-	constructor: WebGL1Renderer,
+	}
 
-	isWebGL1Renderer: true
+}
 
-} );
+WebGL1Renderer.prototype.isWebGL1Renderer = true;
```
Comment (marcofugaro) [inline src/renderers/WebGL1Renderer.js:13]: could you use the `Object.defineProperty()` pattern demonstrated in #20110? 🙏

**Comment by mrdoob** [comment]
Merged #21053 instead. Thanks though!

**Pull request closed without merging**